packages/now-cli/src/util/dev/server.ts

```diff
@@ -780,16 +781,18 @@ export default class DevServer {
 
     // Updating builders happens lazily, and any builders that were updated
     // get their "build matches" invalidated so that the new version is used.
-    this.updateBuildersPromise = updateBuilders(builders, this.output)
-      .then(updatedBuilders => {
-        this.updateBuildersPromise = null;
-        this.invalidateBuildMatches(nowConfig, updatedBuilders);
-      })
-      .catch(err => {
-        this.updateBuildersPromise = null;
-        this.output.error(`Failed to update builders: ${err.message}`);
-        this.output.debug(err.stack);
-      });
+    setTimeout(() => {
```

Comment:
We need to keep track of this timeout instance, and cancel it in `stop()`.

Reply:
I moved the delay into the `updateBuilders()` function so it should be handled with the returned promise